src/jit/regalloc.cpp

```diff
@@ -5641,7 +5646,7 @@ regMaskTP Compiler::rpPredictAssignRegVars(regMaskTP regAvail)
                 // otherwise we will spill this callee saved registers,
                 // because its uses when combined with the uses of 
                 // other yet to be processed candidates exceed our threshold.
-                totalRefCntWtd = totalRefCntWtd;
+                // totalRefCntWtd = totalRefCntWtd;
```

Comment:
Compiler warning, can be suppressed but I would of thought a compiler would optimize it out anyway?